app/assets/stylesheets/application.scss

```diff
@@ -18,8 +18,7 @@
 @import "components/review";
 
 @import "star-rating.js/dist/star-rating.css";
-@import 'tom-select/dist/css/tom-select.css';
```

Comment:
!!! Tom select breaks Heroku. 
I changed it last time to
@import 'tom-select/src/scss/tom-select.scss';
Keep that one when you solve conflicts, otherwise it will break.

Reply:
ok